### PR TITLE
Add visx icon

### LINKED ISSUE #8623: Request: visx
### Brand Name

visx

### Website

https://airbnb.io/visx/

### Popularity Metric

similarweb rank: 526,370
visx is a collection of expressive, low-level visualization primitives for React. Built on top of D3.js. Made by Airbnb.
https://github.com/airbnb/visx/

Readme
License
[MIT license](https://github.com/airbnb/visx/blob/master/LICENSE)
Code of conduct
[Code of conduct](https://github.com/airbnb/visx/blob/master/CODE_OF_CONDUCT.md)
Stars
[17.1k stars](https://github.com/airbnb/visx/stargazers)
Watchers
[148 watching](https://github.com/airbnb/visx/watchers)
Forks
[675 forks](https://github.com/airbnb/visx/forks)

### Official Resources for Icon and Color

https://github.com/airbnb/visx/blob/master/assets/x-hero.png
color: #EBEBEB
background: #E7383C
image

### Additional Comments

here's an svg version of the icon, it has a fill and color, changing fill to none will make it monochrome and changing color will change the monochrome color of the entire thing

visx

here's an alternative version with the background clipped to the 'x' icon

visx-alt

### Contributing

Yes

### PR DESCRIPTION
**Pull request by uncenter**
image

**Issue:** closes #8623.

**Popularity metric:** [18.9k+ stars on GitHub, upward trajectory, recent commits.](https://github.com/airbnb/visx/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -16642,6 +16642,11 @@
             "hex": "007ACC",
             "source": "https://commons.wikimedia.org/wiki/File:Visual_Studio_Code_1.35_icon.svg"
         },
+        {
+            "title": "visx",
+            "hex": "FF1231",
+            "source": "https://airbnb.io/visx/"
```
Comment (adamrusted) [inline _data/simple-icons.json:16648]: I can't see the SVG source here - am I missing something @uncenter?

Comment (uncenter) [reply _data/simple-icons.json:16648]: Yeah it is a bit roundabout to access it:

Screenshot 2024-06-07 at 05 45 26 (Arc)

Comment (adamrusted) [inline _data/simple-icons.json:16648]:
```suggestion
            "source": "https://airbnb.io/visx"
```

**Review by adamrusted — APPROVED** [review]
Single change to the JSON but otherwise this is good to go. I'll get it adapted and merged in. Thanks @uncenter!

**Pull request merged**